src/pages/members/Members.jsx

```diff
@@ -43,34 +47,74 @@ export class Members extends PureComponent {
     }
   };
 
-  enableDeleteMode = (id) => {
-    this.setState({ deleteMode: true, actionUserId: id });
+  setModalMode = (modalMode, actionUserId = null) => {
+    this.setState({ modalMode, actionUserId });
   };
 
-  removeUser = async () => {
+  disableModalMode = () => {
+    setTimeout(() => {
+      this.setState({ modalMode: null, actionUserId: null });
+    }, 300);
+  };
+
+  createUser = async (user) => {
+    try {
+      const { email, password } = await getUserById(auth.currentUser.uid);
+      const newUser = await createUserWithEmailAndPassword(auth, user.email, user.password);
+      await login(user.email, user.password);
+      await sendPasswordResetEmail(auth, user.email, {
+        url: `http://localhost/?uid=${cryptId(newUser.user.uid)}`,
+      });
+      await signOut(auth);
+      await login(email, password);
+      await createUser(newUser.user.uid, user);
+    } catch (error) {
+      console.log(error.message);
+    }
+    this.disableModalMode();
+  };
+
+  updateUser = async (user) => {
     const { actionUserId } = this.state;
-    await deleteUser(actionUserId);
-    this.disableDeleteMode();
+    await updateUser(actionUserId, user);
+    this.disableModalMode();
   };
 
-  disableDeleteMode = () => {
-    setTimeout(() => {
-      this.setState({ deleteMode: false, actionUserId: null });
-    }, 300);
+  removeUser = async () => {
+    const { actionUserId, users } = this.state;
+    const { email, password } = users.find((item) => item.id === actionUserId);
+    try {
+      const currentUser = await getUserById(auth.currentUser.uid);
+      await login(email, password);
+      await deleteUser(auth.currentUser);
+      await signOut(auth);
+      await login(currentUser.email, currentUser.password);
+      await removeUser(actionUserId);
+    } catch (error) {
+      console.log(error);
+    }
+    this.disableModalMode();
   };
 
   render() {
-    const { users, deleteMode } = this.state;
+    const { users, modalMode, actionUserId } = this.state;
+    const actionUser = users.find((item) => item.id === actionUserId);
 
     return (
       <div>
-        <PageHeader text={PAGE_TITLES.members} />
+        <PageHeader text={PAGE_TITLES.members} onClick={() => this.setModalMode(MODAL_MODES.create)} />
```

Comment:
create handler

Reply:
```suggestion
<PageHeader text={PAGE_TITLES.members} onClick={this.setModalMode} />
```
And in setModalMode I set it as default parametr
```suggestion
setModalMode = (modalMode = MODAL_MODES.create, actionUserId = null) => {
    this.setState({ modalMode, actionUserId });
  };
```
Is it ok?